client/src/components/NavBar/index.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+import "./navbar.css";
+
+const NavBar = () => (
+  <div className="navbar-style">
+    <img
+      src="./assets/imgs/flight.png"
+      className="cus-plane-image"
+      alt="plane"
+    />
+    <div className="cus-app-name-style">App Name</div>
+  </div>
+);
+
+export default NavBar;
```

Comment:
I hate to have annoying name change suggestions, but if you find all of the different Nav components getting a little bit confusing, it might help to name them more specifically, like CustomerPageNavBar and so on. 😀

Reply:
Updated.